contracts/modules/capital/MCR.sol

```diff
@@ -73,12 +72,18 @@ contract MCR is IMCR, MasterAware {
       return;
     }
 
+    // copy over values
     mcrFloor = previousMCR.mcrFloor();
-
     mcr = previousMCR.mcr();
     desiredMCR = previousMCR.desiredMCR();
     lastUpdateTime = previousMCR.lastUpdateTime();
+
+    // copy over parameters
+    mcrFloorIncrementThreshold = previousMCR.mcrFloorIncrementThreshold();
```

Comment:
we should add to the v2 fork test to validate all values are moved correctly.

We will be upgrading MCR.sol to get the new logic for the MCR gearing active cover amount computation.